cmake/ycm-0.3.1/find-modules/FindZFP.cmake

```diff
@@ -27,27 +27,28 @@
 include(FindPackageHandleStandardArgs)
 include(SelectLibraryConfigurations)
 
-find_path(ZFP_INCLUDE_DIR
+find_path(ZFP_INCLUDE_DIRS
           NAMES zfp.h
-          PATHS $ENV{ZFP_ROOT}/inc
+          PATHS $ENV{ZFP_DIR}/include
```

Comment:
If there is no reason for not supporting both, I don't see why not. Anyway in this way, if a user has the old version, cmake will not find the library but the user will not have a feedback on the reason...